### PR TITLE
AC-445: Removed redundant variables

### PR DESCRIPTION
**Pull request by ivary43**
I removed all the local reduntant variables  .Please have a look @Sub6Resources :). 
Issue Link : https://issues.openmrs.org/browse/AC-445

### CODE REVIEW

**Comment by Sub6Resources** [comment]
@ivary43 can you move these two commits to separate pull requests? 😃

**Comment by ivary43** [comment]
@Sub6Resources  i have removed the previous commit ,the changes present in it are as per issue .Please check :).

**Comment by ivary43** [comment]
@f4ww4z , @Sub6Resources i have sqaushed the changes .

**Comment by dkayiwa** [comment]
@ivary43 can you look at the merge conflicts?

**Comment by ivary43** [comment]
@dkayiwa  i will do that for sure.

**Comment by ivary43** [comment]
@dkayiwa  done.

**Comment by Sub6Resources** [comment]
@ivary43 could you squash the commits again?

**Comment by ivary43** [comment]
@Sub6Resources  how to squash two non-consecutive commits.

**Comment by Sub6Resources** [comment]
@ivary43 take a look at https://stackoverflow.com/questions/3921708/how-do-i-squash-two-non-consecutive-commits

File: openmrs-client/src/main/java/org/openmrs/mobile/models/Encountercreate.java [hunk 1]
```diff
@@ -136,7 +136,7 @@ public void setObslist()
         this.obslist = gson.toJson(observations,obscreatetype);
     }
 
-    public void pullObslist() {
+    public void pullObslist() {
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/models/Encountercreate.java]: @ivary43 What is the change here?

Comment (ivary43) [reply openmrs-client/src/main/java/org/openmrs/mobile/models/Encountercreate.java]: @f4ww4z  i will do the changes in 2-3 days...actually i have exam going on.

**Pull request merged**